src/services/config.js

```diff
@@ -15,14 +15,21 @@ class ConfigDataProfile {
 
 class ConfigData {
   constructor() {
+    this.edge = process.env.TWILIO_EDGE;
```

Comment:
I think this will stick the current value of the env var in their config file if it gets saved. Better to default to `null` or `undefined`.